docs/en/changelog.md

```diff
@@ -1,5 +1,43 @@
 ## Changelog
 
+### v1.0.0rc2 (1/5/2022)
+
+#### Highlights
+
+- Support spconv 2.0
+- Support MinkowskiEngine with MinkResNet
+- Support training models on custom datasets with only point clouds
+- Update Registry to distinguish the scope of built functions
+- Replace mmcv.iou3d with a set of bird-eye-view (BEV) ops
+
+#### New Features
+
+- Add loader arguments in the configuration files (#1388)
+- Support spconv 2.0 with a solution for compatibility with cuda 9.0 (#1421)
+- Support MinkowskiEngine with MinkResNet (#1422)
+
+#### Improvements
+
+- Add the documentation for model deployment (#1373, #1436)
+- Add Chinese documentation for speed benchmark (#1379), LiDAR-based 3D detection (#1368), 3D segmentation (#1420) and coordinate system refactoring (#1384)
+- Support training models on custom datasets with only point clouds (#1393)
+- Replace mmcv.iou3d with a set of bird-eye-view (BEV) ops (#1403, #1418)
+- Update Registry to distinguish the scope of built functions (#1412, #1443)
+- Replace recommonmark with myst_parser for documentation rendering (#1414)
+
+#### Bug Fixes
+
+- Fix the show pipeline in the [browse_dataset.py](https://github.com/open-mmlab/mmdetection3d/blob/master/tools/misc/browse_dataset.py) (#1376)
+- Fix missing __init__ files after coordinate system refactoring (#1383)
+- Fix the incorrect yaw in the visualization caused by coordinate system refactoring (#1407)
+- Fix NaiveSyncBatchNorm1d and NaiveSyncBatchNorm2d to support non-distributed cases and more general inputs (#1435)
```

Comment:
NaiveSyncBatchNorm1d and NaiveSyncBatchNorm2d -> `NaiveSyncBatchNorm1d` and `NaiveSyncBatchNorm2d`